web/src/games/bombsAndBunnies/components/board.tsx

```diff
@@ -0,0 +1,259 @@
+      this.getPlayerBettingOptions(3),
+      this.getOtherPlayerHandPenalty(4),
+    ];
+  }
+
+  getBetButtons(key: number) {
+    const playerID = this.getBrowserPlayer();
+
+    return (
+      <div key={key} style={{ marginBottom: '-10px' }}>
+        <span style={{ marginRight: '20px' }}>
+          <BetButton
+            click={canBet(this.props.G, this.props.ctx, playerID) ? this._toggleBetPanel.bind(this) : null}
+            active={this.betPanelToggle}
+          ></BetButton>
+        </span>
+        <SkipButton
+          click={canSkipBet(this.props.G, this.props.ctx, playerID) ? this._skipBet.bind(this) : null}
+        ></SkipButton>
+      </div>
+    );
+  }
+
+  getPlayerHand(key: number) {
+    const playerID = this.getBrowserPlayer();
+
+    if (playerID === null) {
+      return (
+        <div key={key} className={css.spectator}>
+          <span>You are in spectator mode.</span>
+        </div>
+      );
+    }
+
+    if (this.betPanelToggle) return null;
+    var player = getPlayerById(this.props.G, playerID);
+
+    return (
+      <PlayerHand
+        key={key}
+        playerId={player.id}
+        hand={player.hand}
+        cardStyle={player.cardStyle}
+        selectCard={canPlaceCard(this.props.ctx, playerID) ? this._selectCard.bind(this) : null}
+      />
+    );
+  }
+
+  getPlayerBettingOptions(key: number) {
+    const playerID = this.getBrowserPlayer();
+
+    if (!this.betPanelToggle) return null;
+
+    return (
+      <BetPanel
+        key={key}
+        bet={this._bet.bind(this)}
+        minBet={this.props.G.minBet}
+        maxBet={this.props.G.maxBet}
+        playerIndex={parseInt(playerID)}
+      />
+    );
+  }
+
+  getOtherPlayerHandPenalty(key: number) {
+    var playerId = this.getBrowserPlayer();
+    var penaltyPlayerId = this.props.G.failedRevealPlayerId;
+    if (!penaltyPlayerId) return null;
+
+    var penaltyPlayer = getPlayerById(this.props.G, penaltyPlayerId);
+    if (!canDiscard(this.props.G, playerId)) return null;
+
+    return (
+      <PlayerHandPenalty
+        key={key}
+        hand={penaltyPlayer.hand}
+        playerId={playerId}
+        cardStyle={penaltyPlayer.cardStyle}
+        targetPlayerId={penaltyPlayerId}
+        selectCard={this._selectPenaltyCard.bind(this)}
+      ></PlayerHandPenalty>
+    );
+  }
+
+  getPlayerZones(key: number) {
+    var currentPlayerId = this.props.ctx.currentPlayer;
+    return (
+      <PlayerZones
+        key={key}
+        betDisplayProps={this.getBetDisplayProps()}
+        discardPileProps={this.getDiscardPileProps()}
+        currentPlayerId={currentPlayerId}
+        perspectivePlayerId={this.getBrowserPlayer()}
+        players={this.props.G.players}
+        revealCard={isRevealing(this.props.ctx) ? this._revealCard.bind(this) : null}
+        canRevealTargetStack={(targetPlayerId: string) =>
+          canRevealTargetStack(this.props.G, this.props.ctx, targetPlayerId)
+        }
+      ></PlayerZones>
```

Comment:
You can replace `></PlayerZones>` with `/>`

Reply:
Thanks! That's handy to know